client/src/pages/LandingPage.jsx

```diff
@@ -14,34 +14,50 @@ function LandingPage({
   favoriteIDs,
   getCurrentPosition,
 }) {
+  const [sortedStations, setSortedStations] = useState(stations);
+  const [isManuallySorted, setIsManuallySorted] = useState(false);
 
+  function sortByPrice() {
+    const copy = [...stations];
```

Comment:
"copy" is not a good name, especially in line 24 one have to figure out what "copy" is. Use something like "stationsToSort", "stationsToBeSorted" or similar.

Reply:
Thanks for the hint